src/upload/types.rs

```diff
@@ -56,7 +56,11 @@ pub struct Response {
     /// Width of the image in pixels (Only for Images)
     pub width: Option<u64>,
     /// In the case of an image, a small thumbnail URL.
-    pub thumbnail_url: String,
+    /// Since the API has not normalized some fields such as thumbnail
+    /// A provided workaround is to attempt the parsing from "thumbnail_url" or "thumbnail"
+    /// CHECK: Could this incur in repeating the same proccess twice so it may have a performance impact?
+    #[serde(alias = "thumbnail")]
+    pub thumbnail_url: Option<String>,
```

Comment:
This would introduce breaking changes to current version, is this actually required?
Perhaps we could discuss it in a separate PR? Given that this is not related to fetching file details?

Reply:
> This would introduce breaking changes to current version, is this actually required?

As long as I've tested the two endpoints which we provided functionality it happened to not use the exactly the same fields. Meanwhile the upload endpoint uses _thumbnail_url_, the details endpoint uses _thumbnail_ hence I tried to make the response structure more general avoiding problems when deserializing the JSON. However, as you said, it'd better to discuss deeper this topic in a separate PR